TYPO3.Neos/Resources/Public/JavaScript/Content/Inspector/InspectorController.js

```diff
@@ -311,6 +333,9 @@ define(
 				cleanProperties = {},
 				enableTransactionalInspector = true;
 
+			this.set('registeredEditors', Ember.Object.create());
+			this.set('listeners', Ember.Object.create());
```

Comment:
This causes issues because it resets the configured listeners again - and it runs after the listeners have been set up in the line prior to line 166. Since that will always result in the configred listeners _or_ and empty Ember.Object, I think this line here can be removed.